brigadier/projects/templates/project_detail.html

```diff
@@ -3,7 +3,7 @@
 {% load i18n %}
 
 {% block container %}
-    <h1>{{ project.project_name }}</h1>
+    <h1>{{ object.project_name }}</h1>
```

Comment:
why do you changed context variable name from project to object?

Reply:
i've deleted model from view. And there wasn't variable project in my template as default.

Reply:
its because you've broken something